osbs/api.py

```diff
@@ -654,38 +666,29 @@ def _get_flatpak_labels(self, repo_info):
             utils.Labels.LABEL_TYPE_COMPONENT: module.name
         }, None
 
-    def _do_create_prod_build(self, git_uri, git_ref,
-                              git_branch,
-                              user,
-                              component=None,
-                              target=None,
-                              yum_repourls=None,
-                              koji_task_id=None,
-                              scratch=None,
-                              platform=None,
-                              platforms=None,
-                              build_type=None,
-                              release=None,
+    # Gives flexmock something to mock
+    def get_user_params(self, req_labels=None, **kwargs):
+        req_labels = req_labels or {}
+        user_params = BuildUserParams(build_json_store=self.os_conf.get_build_json_store())
+        user_params.set_params(build_conf=self.build_conf,
+                               component=req_labels[utils.Labels.LABEL_TYPE_COMPONENT],
+                               name_label=req_labels[utils.Labels.LABEL_TYPE_NAME],
+                               **kwargs)
+        return user_params
+
+    def _do_create_prod_build(self,
+                              git_uri, git_ref, git_branch,
                               inner_template=None,
                               outer_template=None,
                               customize_conf=None,
                               arrangement_version=None,
-                              filesystem_koji_task_id=None,
-                              koji_upload_dir=None,
-                              is_auto=False,
-                              koji_parent_build=None,
-                              isolated=None,
-                              flatpak=False,
-                              signing_intent=None,
-                              compose_ids=None,
-                              reactor_config_override=None,
-                              parent_images_digests=None,
+                              base_image=None,
```

Comment:
`arrangement_version` and `base_image` are unused (or redefined without usage)

Reply:
Thanks!